src/core/agent/services/ipexCommunicationService.types.ts

```diff
@@ -0,0 +1,12 @@
+interface CredentiasMatchingApply {
+  schema: {
+    name: string;
+    description: string;
+  } | null;
+  credentials: {
+    connectionId?: string;
```

Comment:
`connectionId` should not have `?`

Reply:
I see that now the connectionId is not created when creating the ACDC, and in the CredentialMetadata now.
![image](https://github.com/cardano-foundation/cf-identity-wallet/assets/142210850/71f14a14-142d-4f18-a1b5-af0a85067633)

Reply:
Hmm, let me create a ticket to resolve